recipes/napari-organoid-counter/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "napari-organoid-counter" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/napari-organoid-counter-{{ version }}.tar.gz
+  sha256: 6a596bbbcf2653334ffbdee8bd0303b39549d7223d194a774ee3292db1b3e31a
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python >=3.8,<3.10
+    - setuptools-scm
+  run:
+    - napari >=0.4.15
+    - napari-aicsimageio >=0.6.1
+    - python >=3.8,<3.10
+
+test:
+  imports:
+    - napari_organoid_counter
+  commands:
+    - pip list
+    # - pip check  # Fails due to aicsimageio deps and fixed constraintes that were relaxed
+  requires:
+    - pip
+    - pyqt
```

Comment:
Why is this listed here and not in run?

Reply:
@xhochy  napari no longer enforces a qt bidning by default, so when doing

`conda install napari -c conda-forge` or `pip install napari` you will still need to do

`install pyqt` or `install pyside` 


See: https://napari.org/stable/tutorials/fundamentals/installation.html


<img width="1002" alt="Screenshot 2023-10-23 at 5 35 20 PM" src="https://github.com/conda-forge/staged-recipes/assets/3627835/6033b36b-314f-4121-a0b2-17a85fceca08">

Reply:
Ah! The best way forward would be to have a metapackage napari-qt (or a more general name without using napari) which builds two variants, one for each qt binding and let all napari recipes depend on this metapackage if they want qt.

Reply:
This was already discussed and voted against. 

See:
- https://github.com/conda-forge/napari-feedstock/pull/48
- https://github.com/conda-forge/napari-feedstock/issues/47

The napari team is ok with having this as is for both conda and pip.

Reply:
Not really what I think is best, but also not an area where I have something at stake, so let's merge this.